__tests__/app.test.js

```diff
@@ -38,4 +39,17 @@ describe("GET", () => {
         .expect(404);
     })
   });
+  describe('/api', () => {
+    test('STATUS - 200, should respond with all the list of endpoints available ', () => {
+        return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({body}) => {
+          console.log(body)
+          const bodyLength = Object.keys(body).length
+          expect(bodyLength).toBe(3)
+          expect(body).toEqual(endpoints)
```

Comment:
body toEqual the endpoints json is the better way here, there is no need for line 49 or 50. If you're worried its too short, don't be.